HOWTO-make-a-release.md

```diff
@@ -288,7 +288,10 @@ inserting the `-n` (dry-run) option.
 *For OpenSSL before 3.0*, simply push your local changes to the main source
 repo, and please do remember to push the release tags as well, which is done
 separately with the `--tags` option.  You may want to sanity check the
-pushes by inserting the `-n` (dry-run) option.
+pushes by inserting the `-n` (dry-run) option. You must specify the tag to be
+pushed when using `--tags` to avoid pushing all tags in your local repository:
+
+    git push --tags <tagname>
```

Comment:
Actually no.  All you do is push the tag, the same way as you would push a branch.  The easiest way to do that without having to have a registered remote name is this:

    git push openssl-git@git.openssl.org:openssl.git $tag

Reply:
I think we should assume a remote is present. The syntax is overly complex for my liking. I've updated the instructions accordingly.

Reply:
Ok, but you could at least make a proper change:

    git push <remote> <tagname>

...  or actually, if you follow git parlance, it should be this:

    git push <repository> <tagname>

(a "remote" is just a mnemonic for a registered repository)

Reply:
> Ok, but you could at least make a proper change:

Hmmm...I did...but somehow managed to not add it to the fixup...

Reply:
Done now.